src/main/java/net/uaznia/lukanus/hudson/plugins/gitparameter/GitParameterDefinition.java

```diff
@@ -271,8 +278,13 @@ public int compareTo(GitParameterDefinition pd) {
         }
         for (Branch branch : gitClient.getRemoteBranches()) {
             String branchName = branch.getName();
-            if (branchFilterPattern.matcher(branchName).matches()) {
-                branchSet.add(branchName);
```

Comment:
You can't use branch like "origin/master" as branch parameter of Repo SCM.
So I add support of regex group when using branch filter.
If you set branch filter like "origin/(.*)", the branch will like "master", without prefix "origin/", then you can use the branch with Repo SCM.